cfg/credentials.go

```diff
@@ -65,24 +91,26 @@ func GetAPIKeyAndSecret() (string, string) {
 
 // Writes API key ID and secret to the config file.
 func WriteAPIKeyAndSecret(profile, keyID, keySecret string) error {
-	if profile != "default" {
-		return errors.Errorf("non-default profile not supported yet")
+	keyValueMap := map[string]string{
+		"api_key_id":     keyID,
+		"api_key_secret": keySecret,
 	}
 
-	// Create the config file if it doesn't exist.
-	credsFile := GetCredentialsConfigPath()
-	if _, err := os.Stat(credsFile); os.IsNotExist(err) {
-		// Create initial config file.
-		if f, err := os.OpenFile(credsFile, os.O_CREATE|os.O_EXCL, 0600); err != nil {
-			return errors.Wrapf(err, "failed to create %s", credsFile)
-		} else {
-			f.Close()
-		}
-	} else if err != nil {
-		return errors.Wrapf(err, "failed to stat %s", credsFile)
+	return writeConfigToFile(profile, keyValueMap)
+}
+
+// Get Postman API key and environment from config file
+func GetPostmanAPIKeyAndEnvironment() (string, string) {
+	// Only support default profile for now.
+	return creds.GetString("default.postman_api_key"), creds.GetString("default.postman_env")
```

Comment:
Please bind these to environment variables like the akita api key:

```
	creds.BindEnv("default.api_key_id", "AKITA_API_KEY_ID")
	creds.BindEnv("default.api_key_secret", "AKITA_API_KEY_SECRET")
```

Reply:
This is done in other PRs